packages/react-instantsearch-core/src/connectors/connectScrollTo.js

```diff
@@ -67,10 +66,10 @@ export default createConnector({
     // using ScrollTo in combination of Pagination. As pagination can be change
     // by every widget, we want to scroll only if it cames from the pagination
     // widget itself. We also remove the configure key from the search state to
-    // do this comparaison because for now configure values are not present in the
+    // do this comparison because for now configure values are not present in the
     // search state before a first refinement has been made and will false the results.
     // See: https://github.com/algolia/react-instantsearch/issues/164
-    const cleanedSearchState = omit(omit(searchState, 'configure'), id);
```

Comment:
I don't know why this used to call `omit` twice before, it seemed useless there

Reply:
Right, like you did, `['configure', id]` will do the job.

Reply:
We don't have to use `omit` we can leverage the spread operator. It might be easier to read with the function though.